### ISSUE TITLE
Please release 6.0.0 Final when possible

### ISSUE THREAD

**Issue by scottmarlow** [eclipse-ee4j/expressly#19]
I heard that Hibernate Validator 9 would like to include the final release of Expressly 6.0.0 (when that is available).  

We tried updating to 6.0.0-M2-RELEASE but that wasn't on maven central as best as we could tell (could only see https://repo1.maven.org/maven2/org/glassfish/expressly/expressly/6.0.0-M1).  We really would like to update to https://repo1.maven.org/maven2/org/glassfish/expressly/expressly/6.0.0 (e.g. final) when possible so created this issue as a reminder.  Thank you!

**Comment by marko-bekhta** [comment]
Hello @arjantijms ,
Would you happen to have any news on this one? Thanks!

**Comment by arjantijms** [comment]
We're still hoping that one day this one will get back up again:

https://ci.eclipse.org/expressly

It's been down for weeks now, with no telling when (if ever) it will ever get up again. Last week I created a gitlab issue for this, but also no response there.

https://gitlab.eclipse.org/eclipsefdn/helpdesk/-/issues/6139

**Comment by marko-bekhta** [comment]
Ohhh, I see, Expressly also got affected by that CI outage ... 
But otherwise it's ready to go, right?
And thanks for getting back to me on this 😃 👍🏻

**Comment by arjantijms** [comment]
Yes, everything got affected by that. Before there was a release review that took a very long time. It's technically ready to go indeed (and has been for months really)

**Comment by arjantijms** [comment]
The CI is back, and I've staged 6.0.0 (final). Will be running a few short tests, and if all is fine (should be) I'll push it out to central.

**Comment by arjantijms** [comment]
It is done: https://repo1.maven.org/maven2/org/glassfish/expressly/expressly/6.0.0/

Expression Language (EE) TCK tested against GlassFish 8.0.0-SNAPSHOT with Expressly 6.0.0.

```
[INFO] 
[INFO] Results:
[INFO] 
[INFO] Tests run: 717, Failures: 0, Errors: 0, Skipped: 0
```

**Comment by marko-bekhta** [comment]
Nice, thank you!
